### PR TITLE
Converted to Swift 4.2 and Xcode 10

### PR DESCRIPTION
**Pull request by aaronsky**

### CODE REVIEW

File: Example/Source/Navigation/NavigationDetailViewController.swift [hunk 1]
```diff
@@ -53,7 +53,7 @@ class NavigationDetailViewController: BrickViewController {
     }
 
     fileprivate func configure(twoLabel: TwoLabelView, cell: GenericBrickCell) {
-        cell.edgeInsets = UIEdgeInsetsMake(10, 10, 10, 10)
+        cell.edgeInsets = UIEdgeInsets.init(top: 10, left: 10, bottom: 10, right: 10)
```
Comment (jay18001) [inline Example/Source/Navigation/NavigationDetailViewController.swift]: Can we remove the `init`

File: Example/Source/Theme.swift [hunk 2]
```diff
@@ -25,14 +25,14 @@ class Theme {
...
-            .font: Theme.fontForNavigationTitle
+            NSAttributedString.Key.foregroundColor: Theme.textColorForNavigationTitle,
+            NSAttributedString.Key.font: Theme.fontForNavigationTitle
         ]
 
         // EdgeInsets
-        LabelBrickCell.appearance().edgeInsets = UIEdgeInsetsMake(10, 10, 10, 10)
-        ButtonBrickCell.appearance().edgeInsets = UIEdgeInsetsMake(10, 10, 10, 10)
-        SegmentHeaderBrickCell.appearance().edgeInsets = UIEdgeInsetsMake(10, 10, 10, 10)
+        LabelBrickCell.appearance().edgeInsets = UIEdgeInsets.init(top: 10, left: 10, bottom: 10, right: 10)
```
Comment (jay18001) [inline Example/Source/Theme.swift]: Can we remove the init

File: Example/Source/Navigation/NavigationDetailViewController.swift (hunk 1 above)
Comment (aaronsky) [reply Example/Source/Navigation/NavigationDetailViewController.swift]: migrator!!!!!!!!!!

File: Example/Source/Theme.swift (hunk 2 above)
Comment (aaronsky) [reply Example/Source/Theme.swift]: !!!!!!!!!!!!!!!!!!!!!!!!

**Comment by jay18001** [comment]
lgtm

**Pull request merged**